h2o-core/src/main/java/hex/Model.java

```diff
@@ -2380,6 +2381,9 @@ public boolean testJavaScoring(Frame data, Frame model_predictions, EasyPredictM
                   .setEnableStagedProbabilities(genmodel instanceof SharedTreeMojoModel)
                   .setUseExternalEncoding(true); // input Frame is already adapted!
           epmw = new EasyPredictModelWrapper(config);
+          if (genmodel instanceof GamMojoModelBase) {
+            ((GamMojoModelBase) genmodel).setPredictColumnName(genmodel._names.length); // column names found in input and original model
```

Comment:
this looks like poor design, basically what you are doing is genmodel.setPredictColumnName(genmodel._names.length)
1) telling genmodel object about something it already knows about
2) set*Name(.length) is confusing naming

Reply:
This is needed because I need to know what the input row column names are.  In this case, it is use the genmodel._names.  However, in other case, it can be something very different.  I did change a bit to make it clearer.